### DISCUSSION TITLE
Off-screen Carla server built from source

### DISCUSSION THREAD

**Discussion by Pilex1** [carla-simulator/carla#4617] [Q&A]
I've got a Carla server built from source. I'd like to run it off-screen but there doesn't seem to be documentation describing how to do this for a Carla server built from source. The closest I could find is  https://carla.readthedocs.io/en/0.9.12/adv_rendering_options/
which tells me to run `./CarlaUE4.sh -RenderOffScreen`
but there is no CarlaEU4.sh script in the source code.

**Comment by qhaas** [comment]
Try the [`make package`](https://carla.readthedocs.io/en/0.9.12/build_linux/#other-make-commands) target, it will create a tar.gz similar to what you download from the vendor.  The `CarlaUE4.sh` script will get generated at some point when completing that build target.
